client/cypress/integration/app.spec.ts

```diff
@@ -5,28 +5,13 @@ const page = new AppPage();
 describe('App', () => {
   beforeEach(() => page.navigateTo());
 
-  it('Should have the correct title', () => {
-    page.getAppTitle().should('contain', 'Handy Pantry');
+  it('Should have a working toolbar', () =>{
+    cy.get('.toolbar-item').should('contain.text', 'My Pantry');
+    it('Should be able to go to the pages',() =>{
+      page.getNavLink('Products').click();
+      cy.url().should('match', /\/products$/);
+      page.getNavLink('Pantry').click();
+      cy.url().should('match', /^https?:\/\/[^\/]+\/?$/);
```

Comment:
This link is weird, and doesn't really say anything except that there will be a URL there. Would it make more sense to say that the URL should _not_ contain the word "pantry" since we're using `/` as the pantry entry?

Also, would it be reasonable to include a check that clicking on "Shopping List" takes you to a URL containing "shoppinglist"?